incompat/any2stringadd-conversion/src/main/scala/any2stringadd-conversion.scala

```diff
@@ -0,0 +1,3 @@
+object Test {
+  val str = (new AnyRef).toString + "foo"
```

Comment:
This change is not entirely equivalent *in general*. If the object happens to be `null` (not possible in this case but in general yes) then the call to `.toString` will throw an NPE whereas the original code would use the string `"null"`. The best rewrite is to use `"" + ...`:
```suggestion
  val str = "" + new AnyRef + "foo"
```